app/migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
All the code in the `migrations` directory is auto-generated by alembic and executed by running `flask db upgrade`.